packages/neuron-ui/src/components/MultisigAddress/hooks.ts

```diff
@@ -26,90 +27,111 @@ export const useSearch = (clearSelected: () => void) => {
 
   const onSearch = useCallback(
     value => {
-      setSearchKeywords(value)
+      onFilterConfig(value)
       clearSelected()
     },
-    [setSearchKeywords, clearSelected]
+    [onFilterConfig, clearSelected]
   )
 
   const onClear = useCallback(() => {
     onSearch('')
   }, [onSearch])
-  return { keywords, onKeywordsChange, setKeywords, onSearch, searchKeywords, onClear }
+  return { keywords, onKeywordsChange, setKeywords, onSearch, onClear }
 }
 
 export const useConfigManage = ({ walletId, isMainnet }: { walletId: string; isMainnet: boolean }) => {
-  const [configs, setConfigs] = useState<MultisigConfig[]>([])
+  const [entities, setEntities] = useState<MultisigEntity[]>([])
   const saveConfig = useCallback(
-    ({ m, n, r, addresses, fullPayload }) => {
+    ({ m, n, r, addresses }: { m: number; n: number; r: number; addresses: string[] }) => {
       return saveMultisigConfig({
         m,
         n,
         r,
-        addresses,
-        fullPayload,
+        blake160s: addresses.map(v => addressToScript(v).args),
```

Comment:
`addresses` has been validated before saving.

Reply:
> `addresses` has been validated before saving.

This validation assumes the address follows `secp256k1/blake160` lock script, but it could be a different lock script in fact.